### PR TITLE
Expose aggregation method in ensemble NBEATS, fix forecast shape

### LINKED ISSUE #2592: Expose NBEATSEnsemble aggregation_method and wrong dimensions
## Description
(A clear and concise description of what the bug is.)
1. NBEATSEnsemblePredictor has the option of selecting aggregation_method, but it is not exposed to the estimator class. This means that there is no natural way of selecting this parameter. It is possible to interpret the forecasts from the ensemble as samples, and calculate the quantiles from an ensemble. This parameter should be exposed to the estimator class.
2. Manually selecting the aggregation_method="none" still leads to a bug, wrong dimension size for SampleForecast. The bug fails silently when evaluated with the Evaluator class, and leads to erroneous metrics.

https://github.com/awslabs/gluonts/blob/784a882b16fc7704e346cb46cbeb33891b565b99/src/gluonts/mx/model/n_beats/_ensemble.py#L171-L196
Issue should be from here, where  output is shape (N, 1, T)

## To Reproduce
(Please provide minimal example of code snippet that reproduces the error. For existing examples, please provide link.)

```python
from gluonts.mx import NBEATSEnsembleEstimator, NBEATSEnsemblePredictor
from gluonts.dataset.repository.datasets import get_dataset

dataset = get_dataset("airpassengers")

training_data, test_gen = split(dataset.train, offset=-36)
model = NBEATSEnsembleEstimator(
    prediction_length=12, freq="M", 
    meta_context_length=[2], meta_bagging_size=1,
    trainer=Trainer(epochs=1)
).train(training_data)

predictor = NBEATSEnsemblePredictor(12, model.predictors, "none")
test_data = test_gen.generate_instances(12)
forecast_it = predictor.predict(test_data.input)
forecast = next(forecast_it)

print(forecast.samples.shape)
```

## Error message or code output
(Paste the complete error message, including stack trace, or the undesired output that the above snippet produces.)

```
(3, 1, 12)
```

Desired dimensionality should be (3, 12)

## Environment
- Operating system:
- Python version:
- GluonTS version: 0.11.6
- MXNet version:

(Add as much information about your environment as possible, e.g. dependencies versions.)

### PR DESCRIPTION
**Pull request by lostella**
*Issue #, if available:* Fixes #2592

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

**Please tag this pr with at least one of these labels to make our release process faster:** BREAKING, new feature, bug fix, other change, dev setup

### CODE REVIEW

**Comment by gorold** [comment]
https://github.com/awslabs/gluonts/blob/17c4018f3af1dbde8f3ac091ab5fcbd7a09160b9/src/gluonts/mx/model/n_beats/_ensemble.py#L192

https://github.com/awslabs/gluonts/blob/17c4018f3af1dbde8f3ac091ab5fcbd7a09160b9/src/gluonts/mx/model/n_beats/_ensemble.py#L194

I believe the above two lines should be ```output = np.median(output, axis=0, keepdims=True)``` instead, since ```output = np.concatenate(output, axis=0)``` results in an (N, T) array, and SampleForecast takes in an (N, T) array even if N = 1. 

Other than that, looks great! Thanks a lot for the fix!

**Review by jaheba — APPROVED** [review]
